test/test_external.py

```diff
@@ -298,6 +299,29 @@ def test_private_space(self):
         except TooManyRequestsError:
             pass
 
+    def test_multiple_spaces_one_private(self):
```

Comment:
I think it might be worth to test calling `Interface` via function in an example where the output is `gr.File`, `gr.Audio`, or `gr.Video`. Just to have coverage of the changes you added to `FileSerializable`.

Reply:
Added a test!